aidial_analytics_realtime/app.py

```diff
@@ -298,7 +302,15 @@ async def on_log_messages(
                 rates_calculator,
             )
         except Exception as e:
-            logging.exception(e)
+            logging.exception(f"Error processing message #{idx}")
+            any_errors = True
+            statuses.append({"status": "error", "error": str(e)})
+        else:
+            statuses.append({"status": "success"})
+
+    status_code = 500 if any_errors else 200
```

Comment:
Is it OK to return an error here?
Wouldn't the log sending system try to re-send us the same batch of logs again in this case?